src/main.cc

```diff
@@ -13,18 +13,47 @@ int main(int argc, char** argv) {
     //! IO
     std::unique_ptr<IO> io(new IO(argc, argv));
 
-    //! Mesh
-    std::unique_ptr<Mesh<3, 8>> mesh(new GMSH<3, 8>());
-
     //! MaterialProperties
     std::shared_ptr<MaterialProperties> material =
         std::make_shared<MaterialProperties>(io->material_properties());
 
+    // const unsigned Tdim = io->dimension();
+    // const unsigned Tvertices = io->nvertices();
+    // unsigned TTdim = io->dimension();
+    // unsigned Tvertices;
+        
+    // int dim = 2;
+    // switch (dim) {
+    //     case 2: {
+    //         const unsigned Tdim = 2;
```

Comment:
The rest of the main should go here and repeat the main for case 3 as well

Reply:
Understood. For now just leave `Tvertices` together? or you want to have `switch` inside `switch`?

Reply:
Isn't `Tvertices` fixed for 2D and 3D?